server/services/xsd/constants.py

```diff
@@ -0,0 +1,30 @@
+"""This module contains constants related to XSD processing.
+"""
+
+import os
+
+TOOL_XSD_FILE = os.path.join(os.path.dirname(__file__), "galaxy.xsd")
```

Comment:
this file should be taken out of the python package and removed from this source package at some point

Reply:
Sure, it is in my TODO list :)